sim/monk/mistweaver/surging_mist.go

```diff
@@ -0,0 +1,44 @@
+package mistweaver
+
+import (
+	"time"
+
+	"github.com/wowsims/mop/sim/core"
+	"github.com/wowsims/mop/sim/monk"
+)
+
+func (mw *MistweaverMonk) registerSurgingMist() {
+	actionID := core.ActionID{SpellID: 116694}
+	chiMetrics := mw.NewChiMetrics(actionID)
+	spellCoeff := 1.8
+
+	mw.RegisterSpell(core.SpellConfig{
+		ActionID:       actionID,
+		SpellSchool:    core.SpellSchoolNature,
+		ProcMask:       core.ProcMaskSpellHealing,
+		Flags:          core.SpellFlagHelpful | core.SpellFlagAPL,
+		ClassSpellMask: monk.MonkSpellSurgingMist,
+		ManaCost: core.ManaCostOptions{
+			BaseCostPercent: 8.8,
+		},
+		Cast: core.CastConfig{
+			DefaultCast: core.Cast{
+				GCD:      core.GCDDefault,
+				CastTime: time.Millisecond * 1500,
+			},
+		},
+
+		DamageMultiplier: 1,
+		ThreatMultiplier: 1,
+		CritMultiplier:   mw.DefaultCritMultiplier(),
+
+		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+			baseHealing := 19630 + spellCoeff*spell.HealingPower(target)
+			//Hardcoded to heal the player for now
+			spell.CalcAndDealHealing(sim, &mw.Unit, baseHealing, spell.OutcomeHealingCrit)
```

Comment:
If you set the `BonusCoefficient` field on the `SpellConfig` to 1.8, then you can simplify this to just
```
spell.CalcAndDealHealing(sim, target, 19630, spell.OutcomeHealingCrit)
```
and the Spell Power contribution will be handled automatically.

>